src/components/ConnectNode/index.tsx

```diff
@@ -7,20 +7,61 @@ import { useAppDispatch, useAppSelector } from '../../store';
 import { authActions } from '../../store/slices/auth';
 import { nodeActions } from '../../store/slices/node';
 import { appActions } from '../../store/slices/app';
+import { Button, Menu, MenuItem } from '@mui/material';
 
 const Container = styled.div`
-  height: 59px;
-  width: 160px;
-  color: black;
+  align-items: center;
```

Comment:
Now that the whole component is a button, please extend Button from Mui instead of a div to get a nice ripple effect on click and also hover effect.
`const Container = styled(Button)`